docs/src/interface.md

```diff
@@ -242,13 +242,16 @@ Then
 2. implement `f(DM::OurDecoratorManifold, arg1, arg2)`
 
 This makes it possible to extend a manifold or all manifolds with a feature or replace a feature of the original manifold.
+
 The [`MetricManifold`](@ref) is the best example of the second case, since the default metric indicates for which metric the manifold was originally implemented, such that those functions are just passed through.
 This can best be seen in the [`SymmetricPositiveDefinite`](@ref) manifold with its [`LinearAffineMetric`](@ref).
 
+A final techical note – if several manifold have similar transparency rules concerning functions from the interface, the last parameter `T` of the [`AbstractDecoratorManifold`](@ref)`{𝔽,T<:`[`AbstractDecoratorType`](@ref)`}`. Can be used to dispatch on different transparency schemes.
```

Comment:
```suggestion
A final technical note – if several manifolds have similar transparency rules concerning functions from the interface, the last parameter `T` of the [`AbstractDecoratorManifold`](@ref)`{𝔽,T<:`[`AbstractDecoratorType`](@ref)`}` can be used to dispatch on different transparency schemes.
```
This is a bit unclear. Do you want to emphasize here that a single manifold type can have several different transparency rules or that several different manifolds can share the same transparency rules?